aes/decryptor.go

```diff
@@ -0,0 +1,122 @@
+package aes
```

Comment:
Since this package is now AES specific, how about defining an `Encryptor` and `Decryptor` interface in the base sops package and implement it here? That would make adding other algorithms easier.